barretenberg/cpp/src/barretenberg/ultra_honk/ultra_verifier.cpp

```diff
@@ -138,6 +138,7 @@ template <typename Flavor> bool UltraVerifier_<Flavor>::verify_proof(const plonk
 
     // If Sumcheck did not verify, return false
     if (sumcheck_verified.has_value() && !sumcheck_verified.value()) {
+        info("UltraVerifier: Sumcheck failed.");
```

Comment:
stray info

Reply:
I'd like to keep this lil guy around for now